### ISSUE TITLE
ELI5 KILT annotation

### ISSUE THREAD

**Issue by carriex** [facebookresearch/KILT#44]
Hi, thanks for creating the dataset! I have two questions regarding to the annotated ELI5 data:

- According to the KILT data format in the main README, the field `output/provenance/meta` contains task-specific data. What does this meta field contain for the ELI5 dataset? Are these the manual annotation for instances with low overlap between passages and answers mentioned in section 4? If so, could you share a bit more on how is this manual annotation performed?
-  I noticed that for some annotation in the field `output/provenance/meta/evidence_span` contain a string "highlight sentence(s) containing evidence, not only the answer" at the end, (for instance, data in ELI5 dev set with id "3atjp2", as well as "49wqfo"). What are the meanings of such string? 

Thank you!

**Comment by fabiopetroni** [comment]
Hi @carriex,

thanks a lot for your message.
The meta field in the ELI5 dev set contains exhaustive information about our annotation campaign, including the span of text that was highlighted by the annotators. Here the full guidelines for the annotation campaign:

<ul>
  <li> Read the passages </li>
  <li> Decide if the passage can answer the question </li>
  <ul>
      <li> If the passage is not relevant, choose <b>"No"</b> <i>(see Example 1)</i></li>
      <li> If the passage contains evidence to fully answers the question, choose <b>"Yes,
              sufficient to answer"</b>
          <i>(see Example 1 and 2)</i> </li>
      <li> If the passage contains evidence to partially answers the question, choose <b>"Yes,
              partial"</b> <i>(see
              Example 3)</i></li>
  </ul>
  <li> If the passage is relevant, highlight the relevant sentences. <u>We are interested in the
          evidence needed to answer the question</u>, not only the answer. </li>
  <li> A set of valid answers (separated with semicolons) are provided for your benefit, however
      please <u>
          base your decision primarily on the question</u> since there could be other valid
      answers (not reported) and
      reported answers could be wrong. We are interested in the evidence needed to provide a valid
      answer for the question. </li>
  <li> Consider the <u>Wikipedia article title and section as additional contextual
          information</u> for your decision
      <i>(see Example 2)</i> </li>
  <li> We consider <b>partial evidence</b> a sentence that contains some evidence that's
      helpful for answering the question, but the answer cannot be fully worked out by this
      sentence by itself.</li>
  <li> Containing one of the answers is not enough for a sentence to be considered partial
      evidence <i>(see Example 1)</i>.</li>
  <li>There could be some typos - please use your common sense to resolve those.</li>
</ul>

I hope this helps!

**Comment by fabiopetroni** [comment]
1. we don't consider partial evidence in KILT. A wikipedia page is added to the output only if we find the majority of annotators indicating that as containing full evidence. So there might be other pages with partial evidence in meta.
2. I don't fully get the question. In meta we report the complete annotation information, including partial evidence annotation and evidence span if available (even if we ignore this information in the evaluation).

**Comment by carriex** [comment]
thanks for getting back so quickly! The answer for the first question makes sense to me.

For the second question, I'm referring to the fields inside `output/provenance`. For the two examples above, the question with id `1kiwfx` doesn't contain an evidence span in `output/provenance`, but only `start_character`, `end_character`, `start_paragraph` and `end_paragraph`. However for the question with id `3atjp2`, there are two evidence spans inside `output/provenance`, whereas `start_character` and `end_character` contains value `-1`. Does it mean the annotator highlighted an evidence span for `3atjp2`, but selected  "Yes, sufficient to answer" for the passage in `1kiwfx`? 
Sorry for any confusion caused, let me know if this is clear to you!

**Comment by fabiopetroni** [comment]
I see. So sometimes the evidence span is given as offset in a paragraph within the knowledge source, sometimes as a string (probably there the automatic script failed). I hope this helps :)

**Comment by carriex** [comment]
@fabiopetroni got it! thank you so much for your help! :)
